web-local/src/routes/dev/color-palette/+page.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+  import {
+    applyShiftToPalette,
+    getPaletteAndShift,
+  } from "@rilldata/web-common/features/themes/palette-generator";
+  import chroma from "chroma-js";
+  import type { Color } from "chroma-js";
+  import InputV2 from "@rilldata/web-common/components/forms/InputV2.svelte";
+
+  let input: string;
+
+  let inputPalette: Color[];
+  let matchedPalette: Color[];
+  $: if (input) {
+    try {
+      const inputColor = chroma(input);
+      const [palette, shift] = getPaletteAndShift(inputColor);
+      matchedPalette = palette;
+      inputPalette = applyShiftToPalette(palette, shift);
+    } catch (err) {
+      // no-op
+    }
```

Comment:
Can we remove this try/catch? None of these functions seem like they should be fallible. If one is, then we should try to debug it. If the try/catch is absolutely needed because a library function from chroma.js is failing let's remove that dep and use colorio.js